parts/k8s/kubernetesparams.t

```diff
@@ -195,15 +195,6 @@
       },
       "type": "string"
     },
-{{if not IsHostedMaster}}
```

Comment:
Doesn't this section need to stay here to support ACS use cases?

Reply:
This removal is part of the addons refactor. @tariq1890 can verify